qutip/core/operators.py

```diff
@@ -471,6 +472,90 @@ def create(N, offset=0, *, dtype=None):
     return qdiags(data, -1, dtype=dtype)
 
 
+def fdestroy(N, site=0):
```

Comment:
I think `site` can also be a required argument instead of an optional one.

Reply:
yes, thank you for the comment!